tokens/governance/treasury-spend.md

```diff
@@ -1,97 +1,103 @@
 
-It's highly recommended that you preface any proposal with a post on [Moonbeam's Community Forum](https://forum.moonbeam.network){target=\_blank}. You should allow a period of five days for the community to provide feedback on the Moonbeam Forum post. This is especially important for Treasury proposals because of the irrevocable bond required to submit a Treasury spend proposal. You can submit your idea for a Treasury spend proposal by following [the instructions in the Using the Moonbeam Community Forum to Submit a Treasury Proposal guide](https://moonbeam.network/news/using-the-moonbeam-community-forum-to-submit-a-treasury-proposal){target=\_blank}.
+3. **Treasury Approval & Action** - If the Treasury Council agrees, it authorizes the treasury spending and moves the process forward
 
 ![Moonbeam Forum Home](/images/tokens/governance/treasury-proposals/treasury-proposal-1.webp)
 
-## Proposing an Action {: #proposing-an-action }
+## Submitting your Idea to the Forum {: #submitting-your-idea-to-the-forum }
+
+As mentioned in the happy path above, the first step of a treasury proposal is to submit the proposal to the [Moonbeam Forum](https://forum.moonbeam.network/c/governance/treasury-proposals/8){target=\_blank}. To properly submit a treasury spend request, take the following steps:
+
+1. From the [governance section](https://forum.moonbeam.network/c/governance/treasury-proposals/8){target=\_blank}, click **New Topic**. By starting the topic in the governance section, the proposal will come pre-filled with a template to ensure you cover all the necessary points 
+2. Provide a title for the proposal
+3. Enter the contents of the proposal, making sure to cover each of the following categories:
 
-This guide focuses on the mechanics of submitting an on-chain Treasury proposal after you have worked with the Treasury Council to refine your idea. If you haven't yet completed the prior steps of the Treasury proposal process, please take a moment to review the [guidelines of the community Treasury program](https://github.com/moonbeam-foundation/treasury/blob/main/interim/interim_treasury_proposal.md){target=\_blank} and evaluate your standing. Engaging the community and revising your proposal based on feedback received is the most critical piece of your proposal. Submitting the on-chain Treasury proposal is the easy part, demonstrated in the following guide.
+    - **Title** and **Proposal Status** 
 
-To get started, head to [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss://wss.api.moonbase.moonbeam.network#/treasury){target=\_blank} and take the following steps:
+    - **Abstract** - two or three sentences that summarize the MBTP
 
-1. Select the **Governance** heading
-2. Click on **Treasury**
-3. Click **Submit Proposal**
+    - **Motivation** - a statement on why the Moonbeam Community via the Treasury Council should support the MBTP
```

Comment:
```suggestion
    - **Motivation** - a statement on why the Moonbeam community via the Treasury Council should support the MBTP
```

Reply:
Ok, so I just realized that we're just repeating the template. I don't think that's necessary. We're not providing any extra value by using the same exact text that they're already going to see when creating the proposal